etc/lime-elements.api.md

```diff
@@ -2121,6 +2126,8 @@ export interface MenuItem<T = any> {
     text: string;
     value?: T;
 }
+export { MenuItem }
+export { MenuItem as MenuItem1 }

```

Comment:
I don't understand why api-extractor suddenly felt the need to export this interface twice, but I don't think it causes any problems… 🤷‍♂️